CMakeLists.txt

```diff
@@ -195,3 +195,4 @@ enable_testing()
 add_subdirectory(test)
 add_subdirectory(examples)
 add_subdirectory(bench/wtperf)
+add_subdirectory(bench/workgen)
```

Comment:
Is it possible to alphabetize this list or is there a specific order to respect?

Reply:
In lieu of of the greater restyling ticket, I haven't been really enforcing that convention in the CMake sources. Though I've applied it here in this specific case `88652cb `